src/test/java/org/isf/generaldata/GeneralDataTest.java

```diff
@@ -0,0 +1,22 @@
+package org.isf.generaldata;
+
+import static org.junit.Assert.*;
```

Comment:
We are trying not to use package style import statements but rather enumerating them all.

Reply:
replace by 
`import static org.assertj.core.api.Assertions.assertThat;`